README.md

```diff
@@ -25,10 +26,41 @@ $ export REGISTRY=<SOME_REGISTRY> \
 ## Using the Marketplace Operator
 
 ### Description
+The operator manages two CRDs: [OperatorSource](./deploy/crds/operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/crds/catalogsourceconfig.crd.yaml).
 
-The marketplace operator manages two CRDs: [OperatorSource](./deploy/crd/operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/crd/catalogsourceconfig.crd.yaml). When an OperatorSource CR is created in the same namespace as where the marketplace operator is running (we recommend the namespace be called "openshift-marketplace"), the operator will download artifacts stored in the registry specified in this OperatorSource CR (for now, please see documentation about using [quay](https://quay.io)'s appregistry API). For an example of this OperatorSource CR please see the [examples](./deploy/examples/) folder.
+`OperatorSource` is used to define the external datastore we are using to store operator artifacts. At the moment we only support Quay's app-registry as our external datastore. Here is an example `OperatorSource`:
```

Comment:
`we are using to store operator bundles.`

Reply:
That's how operator-registry defines a `bundle` - `We refer to a directory of files with one ClusterServiceVersion as a "bundle".`. An operator artifact may have more than one `csvs`.

Reply:
@tkashem are you requesting that I change to using `artifacts` instead of `bundle`. I think `bundle` seems to be the way OLM is also [referring](https://github.com/operator-framework/operator-registry#manifest-format) to artifacts these days.